storage/users.go

```diff
@@ -9,35 +9,77 @@ import (
 	"github.com/Southclaws/cj/types"
 )
 
-// User is a recorded and verified SA:MP forum user.
+// User is a recorded and verified burgershot forum user.
 type User struct {
-	DiscordUserID    string `json:"discord_user_id" bson:"discord_user_id"`
-	ForumUserID      string `json:"forum_user_id" bson:"forum_user_id"`
-	VerificationCode string `json:"verification_code" bson:"verification_code"`
-	ForumUserName    string `json:"forum_user_name" bson:"forum_user_name"`
+	DiscordUserID string `json:"discord_user_id" bson:"discord_user_id"`
+	ForumUserID   string `json:"forum_user_id" bson:"forum_user_id"`
+	ForumUserName string `json:"forum_user_name" bson:"forum_user_name"`
+	BurgerVerify  bool   `json:"burgershot_verified" bson:"burgershot_verified"`
 }
 
 // StoreVerifiedUser is for when a user finishes their verification.
 func (m *MongoStorer) StoreVerifiedUser(verification types.Verification) (err error) {
 	err = m.accounts.Insert(&User{
-		DiscordUserID:    verification.DiscordUser.ID,
-		ForumUserID:      verification.ForumUser,
-		VerificationCode: verification.Code,
-		ForumUserName:    verification.UserProfile.UserName,
+		DiscordUserID: verification.DiscordUser.ID,
+		ForumUserID:   verification.ForumUser,
+		ForumUserName: verification.UserProfile.UserName,
+		BurgerVerify:  true,
 	})
 
 	return
 }
 
+// SetLegacyUserToVerified is for re-verification when a legacy verified user gets set to verified.
```

Comment:
I think a cleaner API would be `SetUserVerified` with new fields, with the original code being deleted. The concept of "re-verifying" shouldn't really exist, it's just another form of verifying existing alongside the other system.